docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -0,0 +1,56 @@
+[role="xpack"]
+[[alerting-troubleshooting]]
+== Alerting Troubleshooting
+
+Use the information in this section to troubleshoot common problems and find answers for frequently asked questions.
```

Comment:
Here's the content we recommend in our template:

This page describes how to resolve common problems you might encounter with Alerting. If your problem isn’t described here, please review open issues in the following GitHub repositories:

- name of repo

Have a question? Contact us in the [https://discuss.elastic.co/](discuss forum).

Reply:
ah, cool, I'll use that instead.
I copied this from [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/monitoring-troubleshooting.html) to keep things consistent.

Not sure who owns those docs... might be worth pinging them here 🤔